src/modules/licence-import/load/licence.js

```diff
@@ -1,9 +1,21 @@
 const connectors = require('./connectors');
+const config = require('../../../../config');
+const roles = require('../transform/mappers/roles');
+
+const isImportableRole = role => {
+  if (config.import.licences.isBillingDocumentRoleImportEnabled) {
+    return true;
+  }
+  // Allow import of billing document roles to be disabled for charging go live
+  return role.role !== roles.ROLE_BILLING;
```

Comment:
I think I understand what's happening on lines 6 & 7 but not sure what line 10 means. This isn't a problem as such. Just stating that I'm not sure what the desired behaviour is. Might be worth slotting an inline comment? [Not important]

Reply:
Done thanks